### ISSUE TITLE
[Bug]: Slime Chunk detection not working on mulitplayer servers

### ISSUE THREAD

**Issue by von-get** [TreyRuffy/BetterF3#59]
### Describe the Bug

Not sure if this is intended behavior, but Slime Chunk simply states "Unknown" when connected to a fairly vanilla server.

Minecraft: 1.20.1

### Steps to Reproduce

Connect to multiplayer server
Press F3

### Expected Behavior

Learn whether or not a chunk is a slime chunk in multiplayer.

### Modloader Type

Fabric

### Modloader Version

0.15.3

### Mod Version

7.0.2

**Comment by MikolajMNK19** [comment]
BetterF3 can't know whether a chunk is a slime chunk, as it does not know the seed which isn't shared with a client. I've tried a client/serverside mod called [SlimeChunkF3](https://www.curseforge.com/minecraft/mc-mods/slimechunkf3) but it seems to not support BetterF3

**Comment by BioTechproject27** [comment]
Yeah, as the other person said your client has to know the seed for that to work.

Perhaps one could add a feature, like one that [minihud](https://github.com/maruohon/minihud) has, where running /seed manually on a server (and having the server respond with one ofc) or being able to enter a manual seed somewhere in the mods settings, would enable that feature on servers.
